DataService.js

```diff
@@ -13,7 +13,8 @@ angular.module('PathOfDamage')
               total: 0
             },
             more: {
-              placeholder: "More Damage"
+              placeholder: "More Damage",
+              values: []
```

Comment:
you access .values in the damage calculations but you didn't initialize them, so I added it.